app/src/main/java/org/schabi/newpipe/local/bookmark/BookmarkFragment.java

```diff
@@ -98,23 +109,83 @@ protected void initViews(final View rootView, final Bundle savedInstanceState) {
     protected void initListeners() {
         super.initListeners();
 
+        final ImageButton mergeAll = activity.findViewById(R.id.mergeButton);
+        mergeAll.setOnClickListener(v -> {
+            if (selectedRemotePlaylists.isEmpty() && !selectedLocalPlaylists.isEmpty()
+            && selectedLocalPlaylists.size() + selectedRemotePlaylists.size() >= 2) {
```

Comment:
The `size>=2` requirement for merging is not needed, since when `size==1` the buttons acts as a "duplicate playlist" and there is nothing wrong with that.